core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerServiceSpec.js

```diff
@@ -0,0 +1,402 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the SimpleEditorManagerService.js
+ */
+
+describe('Simple Editor Manager Service', function() {
+  var stubs;
+  var data;
+  var mockExplorationData = {
+    explorationId: 0,
+    autosaveChangeList: function() { },
+    discardDraft: function() { }
+  };
+
+  var simpleEditorManagerService;
+  var questionObjectFactory;
+  var explorationStatesService;
+  var explorationTitleService;
+  var explorationInitStateNameService;
+  var statesToQuestionsService;
+  var questionListObjectFactory;
+  var validatorsService;
+  var simpleEditorShimService;
+
+  beforeEach(module('oppia'));
+
+  beforeEach(module(function($provide) {
+    $provide.value('explorationData', mockExplorationData);
+  }));
+
+  beforeEach(inject(function() {
+    GLOBALS.NEW_STATE_TEMPLATE = {
```

Comment:
I thought this was resolved in core/tests/karma-constants.js, which loads the same constants file for the unit tests. @ashish3805 did this not work? If so, @vojtechjelinek, can you clarify?